zapd/models.py

```diff
@@ -161,6 +161,32 @@ def in_status(cls, session, status):
     def __repr__(self):
         return "<Proposal %r>" % (self.id)
 
+class Wallet(db.Model):
+    id = db.Column(db.Integer(), primary_key=True)
+    wallet_address = db.Column(db.String(), unique=True)
```

Comment:
this can just be 'address' (because the table name is 'wallet')

Reply:
changed.